doc/api/esm.md

```diff
@@ -266,6 +266,44 @@ modules it can be used to load ES modules.
 The `import.meta` meta property is an `Object` that contains the following
 properties.
 
+### `import.meta.__dirname`
+
+> Stability: 3 - Legacy: Use `import.meta.url` instead.
+
+* {string} A convenience shortcut that provides the equivalent of the CommonJS
+  `__dirname` pseudo-global. Specifically, it returns the result of converting
+  the `import.meta.url` into a file path then returning the directory portion.
+
+The property is only available when `import.meta.url` specifies a `file://`
```

Comment:
the check in the impl only uses `file:` should you be using `//` in the impl or remove that bit from here to just be `file:`? generally when talking about scheme we just have `scheme:` (E.G. https://nodejs.org/dist/latest-v14.x/docs/api/esm.html#esm_file_urls )